To: vim_dev@googlegroups.com
Subject: Patch 7.3.409
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.409
Problem:    The license in pty.c is unclear.
Solution:   Add a comment about the license.
Files:	    src/pty.c


*** ../vim-7.3.408/src/pty.c	2011-05-19 13:40:47.000000000 +0200
--- src/pty.c	2012-01-20 21:07:56.000000000 +0100
***************
*** 14,19 ****
--- 14,23 ----
   * It has been modified to work better with Vim.
   * The parts that are not used in Vim have been deleted.
   * See the "screen" sources for the complete stuff.
+  *
+  * This specific version is distibuted under the Vim license (attribution by
+  * Juergen Weigert), the GPL applies to the original version, see the
+  * copyright notice below.
   */
  
  /* Copyright (c) 1993
*** ../vim-7.3.408/src/version.c	2012-01-20 20:54:15.000000000 +0100
--- src/version.c	2012-01-20 21:13:11.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     409,
  /**/

-- 
ARTHUR: What are you going to do. bleed on me?
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
